cmake/Modules/FindComputeCpp.cmake

```
@@ -63,7 +63,7 @@ set(COMPUTECPP_USER_FLAGS "" CACHE STRING "User flags for compute++")
 separate_arguments(COMPUTECPP_USER_FLAGS)
 mark_as_advanced(COMPUTECPP_USER_FLAGS)
 
-set(COMPUTECPP_BITCODE "spir64" CACHE STRING
+set(COMPUTECPP_BITCODE "" CACHE STRING
```

```suggestion
set(COMPUTECPP_BITCODE "" CACHE LIST
```

If you make this change, you can actually leave the code on line 188 onwards unchanged, since empty `foreach` loops don't actually execute the body at all.

---

Yeah, that works. The only issue I guess is that it does spit out this warning.
```
CMake Warning (dev) at cmake/Modules/FindComputeCpp.cmake:66 (set):
  implicitly converting 'LIST' to 'STRING' type.
```

---

Going to leave it out. Would rather avoid the warning.

---

CACHE variables cannot be LIST: https://cmake.org/cmake/help/latest/command/set.html#set-cache-entry